### PR TITLE
chore: create the login view

### PR DESCRIPTION
**Pull request by Albert-Byrone**
Create the login view  page

### CODE REVIEW

File: src/views/Home.tsx [hunk 1]
```diff
@@ -1,17 +1,45 @@
...
+          <button className="w-full py-4 mt-8 bg-zinc-900/90 text-white">Sign In</button>
+          <div>
+            <p className="flex items-center mt-2">
+              <input type="checkbox"  className="mr-2"/>
+              Remember Me
+            </p>
+            <p className="text-center mt-8">Create an account</p>
+          </div>
+        </form>
+      </div>
```
Comment (sinatra-ms) [inline src/views/Home.tsx]: Have this sign in page as a separate component on its own.

**Review by sinatra-ms — CHANGES_REQUESTED** [review]
Hello @Albert-Byrone Good Job on the PR. I have left some comments on the Review.

**Review by sinatra-ms — APPROVED** [review]
Hello @Albert-Byrone ,
Everything looks good to me.

**Review by mutuajoseph — APPROVED** [review]
Loving the JSX implemented 👌🏾

**Pull request merged**